pages/dashboard.js

```diff
@@ -63,40 +63,40 @@ import DSFooter from '../components/molecules/DSFooter'
 export default function Dashboard(props) {
   return (
     <>
-      <DSHeader locale="en" />
+      <DSHeader locale={props.locale} langToggleLink={props.langToggleLink} />
       <LayoutContainer>
-        <div className="col-span-12 mb-8">
-          <Greeting locale="en" name="Mary" />
+        <div className="col-span-12">
```

Comment:
I believe we want to keep this class
```suggestion
        <div className="col-span-12 mb-8">
```